src/components/PlantsV2/index.tsx

```diff
@@ -161,9 +163,30 @@ export default function PlantsDashboardV2(): JSX.Element {
   const hasPolygons =
     !!plantingSiteResult && !!plantingSiteResult.boundary && plantingSiteResult.boundary.coordinates?.length > 0;
 
+  const getObservationHectares = () => {
+    let observationMonitoringPlots = 0;
+    latestObservation?.plantingZones.forEach((pz) => {
+      pz.plantingSubzones.forEach((psz) => {
+        observationMonitoringPlots += psz.monitoringPlots.length;
+      });
+    });
```

Comment:
Nice!! we can certainly get this in and iterate as we get confirmation.
Re: code.. another suggestion if you want something shorter/functional
```
const numMonitoringPlots = latestObservation?.plantingZones.flatMap(pz => pz.plantingSubzones.flatMap(psz => psz.monitoringPlots))?.length ?? 0;
```

Reply:
Thanks!! Will use this nicer way. And yes, when I get a confirmation I will iterate if needed